### ISSUE TITLE
support mino acid sequences?

### ISSUE THREAD

**Issue by CTGitbub** [yangao07/abPOA#65]
May I ask if abPOA supports MSA of amino acid sequences

**Comment by yangao07** [comment]
Yes, please refer to the updated [readme](https://github.com/yangao07/abPOA?tab=readme-ov-file#generate-consensus-sequence-for-amino-acid-sequences) for amino acid sequences.
